components/eventing-controller/reconciler/subscription/reconciler_test.go

```diff
@@ -692,12 +692,6 @@ var _ = Describe("Subscription Reconciliation Tests", func() {
 				subscription.Spec.Filter = nil
 				return subscription
 			}()),
-		Entry("protocolsettings missing",
-			func() *eventingv1alpha1.Subscription {
-				subscription := reconcilertesting.NewSubscription("schema-protocolsettings-missing", "", reconcilertesting.WithWebhookAuthForBEB)
-				subscription.Spec.ProtocolSettings = nil
-				return subscription
-			}()),
```

Comment:
This is because protocol-settings is no more mandatory as done [here](https://github.com/kyma-project/kyma/pull/11210/files?file-filters%5B%5D=.yaml#diff-2df5d1e812b22dc574de3ff749b14f1b612b26c9d8bded049233f3849a00ea1dL129).